checker/tests/signedness/CastedShifts.java

```diff
@@ -36,35 +36,25 @@ public void CastedIntShifts(@Unsigned int unsigned, @Signed int signed) {
         byteRes = (@Signed byte) (signed >> 0);
 
         // Cast to char.
-        @UnknownSignedness char charRes;
+        char charRes;
 
         // Shifting right by 23, the introduced bits are cast away
         charRes = (@Unsigned char) (unsigned >>> 23);
         charRes = (@Unsigned char) (unsigned >> 23);
-        charRes = (@Signed char) (signed >>> 23);
-        charRes = (@Signed char) (signed >> 23);
```

Comment:
It looks like all the signed char casts are removed from this test. Should at least one remain and expect an error? Or what is supposed to happen if somebody writes that?

Reply:
It is an error to write `@Signed char`.  We'll add a test later, in pull requests after this one.